core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -97,6 +97,13 @@ private[spark] object BlockManagerMessages {
   case class GetLocationsAndStatus(blockId: BlockId, requesterHost: String)
     extends ToBlockManagerMaster
 
+  case class GetLocalDirs(executorIds: Array[String]) extends ToBlockManagerMaster
+
+  /**
+   * The response message of `GetLocalDirs` request.
+   */
+  case class BlockManagerLocalDirs(localDirs: Map[String, Array[String]])
```

Comment:
Do you need this message? Can't you just return `Map[String, Array[String]]` directly from the RPC?